### PR TITLE
Close #40 Add date picker

### LINKED ISSUE #40: Add date picker
As a guest, 
So that I can select a booking date,
I want to be able to see available dates for the space.

### PR DESCRIPTION
**Pull request by kylebuttner**

### CODE REVIEW

File: app/public/interface.js [hunk 1]
```diff
@@ -1,4 +1,30 @@
 $(document).ready(function() {
+  $(function() {
 
-  
-})
\ No newline at end of file
+      function cb(start, end) {
```
Comment (robota-x) [inline app/public/interface.js]: give this a more explicative name?

File: app/public/interface.js [hunk 2]
```diff
@@ -1,4 +1,30 @@
...
+              "cancelLabel": "Clear"
+          },
+          "isInvalidDate": function(date){
+
+            for(i=0;i<booked_dates.length;i++){
+              if (date.format('YYYY-MM-DD') === booked_dates[i]){
+                return true;
+              }
+            }
+        },
```
Comment (robota-x) [inline app/public/interface.js]: check indentation

File: app/public/main.css [hunk 3]
```diff
@@ -91,3 +91,13 @@ h1 {
 .error {
   background-color: rgb(217, 83, 79);
 }
+
+/* CALENDAR PICKER */
```
Comment (lorenzoturrino) [inline app/public/main.css]: remove comment

File: app/public/main.css [hunk 4]
```diff
@@ -91,3 +91,13 @@ h1 {
...
+
+#reportrange{
```
Comment (robota-x) [inline app/public/main.css]: style: space before {

File: app/views/bookings/new.erb [hunk 5]
```diff
@@ -1,7 +1,15 @@
 <h1>Please select a date for your booking.</h1>
 
 <form action="/bookings" method="post">
-  <input type="text" name="booking_start" placeholder="01/01/2016">
-  <input type="text" name="booking_end" placeholder="01/01/2016">
-  <button type="submit">Submit</button>
+  <input id="booking_start" type="hidden" name="booking_start">
+  <input id="booking_end" type="hidden" name="booking_end">
+  <div id="reportrange">
+      <i class="glyphicon glyphicon-calendar fa fa-calendar"></i>&nbsp;
```
Comment (robota-x) [inline app/views/bookings/new.erb:7]: check this

File: app/views/layout.erb [hunk 6]
```diff
@@ -1,18 +1,26 @@
 <!DOCTYPE html>
 <html>
   <head>
+    <script src="https://cdn.jsdelivr.net/jquery/1/jquery.min.js"></script>
+    <script src="<%=url('interface.js')%>"></script>
     <link href='https://fonts.googleapis.com/css?family=Montserrat' rel='stylesheet' type='text/css'>
     <link href='https://fonts.googleapis.com/css?family=Cardo' rel='stylesheet' type='text/css'>
     <link rel="stylesheet" type="text/css" href="/reset.css"/>
     <link rel="stylesheet" type="text/css" href="/main.css"/>
+    <link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/bootstrap/latest/css/bootstrap.css"/>
```
Comment (robota-x) [inline app/views/layout.erb:9]: require order: bootstrap.css before main.css to avoid overwriting our style @AnnemarieKohler

**Comment by robota-x** [comment]
done commenting, good job!

File: app/public/interface.js (hunk 1 above)
Comment (kylebuttner) [inline app/public/interface.js]: changed to `setRange()`

File: app/public/interface.js (hunk 2 above)
Comment (kylebuttner) [inline app/public/interface.js]: done!

File: app/public/main.css (hunk 4 above)
Comment (kylebuttner) [inline app/public/main.css]: done!

File: app/views/bookings/new.erb (hunk 5 above)
Comment (kylebuttner) [inline app/views/bookings/new.erb:7]: Adds small space after glyph. Going to keep for style.

File: app/views/layout.erb (hunk 6 above)
Comment (kylebuttner) [inline app/views/layout.erb:9]: done!

**Comment by kylebuttner** [comment]
@lorenzoturrino can you merge once you've given it another look?

**Comment by robota-x** [comment]
:heart:

**Pull request merged**